dockerfiles/centos-stream8-vdbench5.04.07-container-disk/install_vdbench.sh

```diff
@@ -19,4 +30,15 @@ yum -y groupinstall "Development Tools"
 tar -xvf ntfs-3g_ntfsprogs-2017.3.23.tgz --strip-components 1
 ./configure --prefix=/usr/local --disable-static
 make
-rm -rf /ntfs3g
\ No newline at end of file
+make install
+rm -rf /ntfs3g
+
+# extracting vdbench files
+unzip /vdbench/vdbench50407.zip -d /vdbench/
+chmod +x /./vdbench/vdbench
+
+# add-on for state-signals
+dnf install -y python3.9
+cd /
+python3.9 -m pip --no-cache-dir install --upgrade pip && pip3.9 --no-cache-dir install state-signals --upgrade
+vi state_signals_responder.py
```

Comment:
What is this doing?

Reply:
Fixed it to: cp state_signals_responder.py /state_signals_responder.py

Reply:
It's still showing up; why is the installation script invoking vi?